packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneAttachmentBlobs.spec.ts

```diff
@@ -334,7 +334,7 @@ describeCompat("GC attachment blob tombstone tests", "NoCompat", (getTestObjectP
 		);
 
 		itExpects(
-			"logs error on retrieval of tombstones attachment blobs when ThrowOnTombstoneUsage is not enabled",
+			"logs error on retrieval of tombstones attachment blobs when ThrowOnTombstoneLoad is not enabled",
```

Comment:
Shouldn't it be logs a generic event (not an error)?  No need to make a change here, it's not a big deal.

Reply:
true. updated the name.